app/models/spree/order_decorator.rb

```diff
@@ -263,7 +263,15 @@ def set_variant_attributes(variant, attributes)
   end
 
   def line_item_variants
-    line_items.map(&:variant)
+    if line_items.loaded?
+      line_items.map(&:variant)
+    else
+      line_items.includes(:variant).map(&:variant)
+    end
+  end
+
+  def insufficient_stock_lines
```

Comment:
can we move it to a scope, for consistency?

Reply:
Ah, this is overriding a spree method. I'll update the commit history to make that clear and make some adjustments here...